src/azul/service/hca_response_v5.py

```diff
@@ -592,7 +598,8 @@ def map_entries(self, entry):
                         donorOrganisms=self.make_donors(entry),
                         organoids=self.make_organoids(entry),
                         cellSuspensions=self.make_cell_suspensions(entry),
-                        **kwargs)
+                        **kwargs,
+                        dates=self.make_dates(entry))
```

Comment:
And here. `kwargs` should always come last anyways.
```suggestion
                        dates=self.make_dates(entry),
                        **kwargs)
```